docs/extending.md

```diff
@@ -0,0 +1,25 @@
+# Extending Kinto.js
+
+## Custom database adapters
+
+By default, Kinto.js performs all local persistence operations using IndexedDB; though if you want to create and use you own, that's definitely possible.
+
+Simply create a class and extends from `Kinto.BaseAdapter`, which rather acts as an interface than anything else here:
+
+```js
+class MyAdapter extends Kinto.BaseAdapter {
+  create(record) {
+    …
+  }
+
+  update(record) {
+    …
+  }
+
+  …
+}
+
+const kinto = new Kinto({adapter: MyAdapter});
```

Comment:
We should say something like: "Then you can create the Kinto object passing you adapter and you are all set."